packages/devtools_app/lib/src/ui/fake_flutter/icons.dart

```diff
@@ -15,4 +15,9 @@ class Icons {
   static const IconData open_in_new = IconData(0xe89e);
   static const IconData refresh = IconData(0xe5d5);
   static const IconData stop = IconData(0xe047);
+  static const IconData pause = IconData(0xe034);
```

Comment:
are these icons in tip of trunk package flutter?

Reply:
Yes they are.